src/node.cc

```diff
@@ -905,12 +905,21 @@ Local<Value> UVException(Isolate* isolate,
 
 
 // Look up environment variable unless running as setuid root.
-inline const char* secure_getenv(const char* key) {
+inline bool secure_getenv(const char* key, std::string* text) {
```

Comment:
If you expose it in node_internal.h, I can drop https://github.com/nodejs/node/pull/11006/commits/7c6a26e5aefc1ef8c113b730e0268525aa7a6c36